src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -6,6 +6,7 @@
  void npe(String a, String b) {
   if (a == null) {
    System.out.println();
+   System.out.println();
```

Comment:
**Reporter**: PMD
**Rule**: EmptyIfStmt
**Severity**: WARN
**File**: src/main/java/se/bjurr/violations/lib/example/MyClass.java L9

Empty Code http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/empty.html#EmptyIfStmt

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-123079271>*